docs/src/tutorial/embedding.rst

```diff
@@ -170,3 +170,23 @@ One of the Cython developers has written a
 However, he considers it sufficiently hacky that it is only linked here,
 and not reproduced directly. It is an option though, if you're unable to
 use the inittab mechanism before initializing the interpreter.
+
+Problems with multiprocessing and pickle
+---------------------------------------
+
+If you try to use ``multiprocessing`` while using a Cython module embedded into
+an executable it will likely fail errors related to the pickle module.
+``multiprocessing`` often uses pickle to serialize and deserialize data to
+be run in another interpreter.  What happens is that it starts a fresh copy
+of the **Python** interpreter (rather than a fresh copy of your embedded program)
```

Comment:
IIRC, it actually uses `fork()` on unixy systems. Program starting is only used on Windows (which lacks `fork()`).